Agents/UserAgent/stack-manager-config/inputs/data/user.obda

```diff
@@ -0,0 +1,21 @@
+[PrefixDeclaration]
+xsd:    http://www.w3.org/2001/XMLSchema#
+bot:    https://w3id.org/bot#
+mon:    https://w3id.org/MON/person.owl#
+sla:    https://www.theworldavatar.com/kg/sensorloggerapp/
+
+[MappingDeclaration] @collection [[
+mappingId   User
+target      mon:person_{id} a mon:Person;
+                mon:personalMailbox {email}^^xsd:string;
```

Comment:
Kok Foong suggested before, that ^^xsd:string is not needed, the system will automatically read as string, this is optional one, no need to do anything this time